Dockerfile

```diff
@@ -1,17 +1,30 @@
-FROM golang:alpine as builder
+FROM golang:1.19.1-alpine as build
+
+ARG TARGETPLATFORM
+RUN echo "Building for ${TARGETPLATFORM}"
+
+WORKDIR /go/src
+
+COPY go.mod go.sum ./
+RUN go mod download
 
-WORKDIR /go/src/github.com/boynux/squid-exporter
 COPY . .
 
-# Compile the binary statically, so it can be run without libraries.
-RUN CGO_ENABLED=0 GOOS=linux go install -a -ldflags '-extldflags "-s -w -static"' .
+RUN CGO_ENABLED=0 go build -a -ldflags '-extldflags "-s -w -static"' -o /squid-exporter .
+
 
-FROM gcr.io/distroless/static:nonroot
-COPY --from=builder /go/bin/squid-exporter /usr/local/bin/squid-exporter
+FROM scratch as final
```

Comment:
Since squid-exporter is a binary, we can use the scratch as a base image.

Reply:
I switched from scratch to distroless a while ago. Distroless base image is pretty light and has a few benefits over the scratch base, namely, directory structure, SSL certs, proper user ids, timezone data, and more. 

In this case I prefer to keep the distroless as it's more future proof and the size tradeoff is insignificant.

Can you revert this build in your PR please?